### PR TITLE
Start balance logic adjusted and copy changes

### PR DESCRIPTION
**Pull request by MissyM2**
For the first week that the user starts with the app, the _Week Starting Balance_ should be the amount of the _starting balance transaction_.  In all other weeks, the _Week Starting Balance_ is the amount calculated in the _@computed get weekStartingBalanceText()_ in the ui-store.js. 

Closing #91 
Closing #92 

## How to test this PR

1.  Open the updated app locally. 
2.  Add a _starting balance_
3. When you arrive at the Calendar View, review the first week and see the _Starting Balance_ on the header of the detail area.  It should be the amount you entered as the _starting balance_.  
4.  View the following copy changes:
Screen Shot 2020-07-03 at 10 33 41 AM

## Screenshots

<p float="left">
Screen Shot 2020-07-03 at 10 21 20 AM
Screen Shot 2020-07-03 at 10 21 28 AM
Screen Shot 2020-07-03 at 10 21 40 AM
</p>
<p>Screen Shot 2020-07-03 at 10 21 50 AMScreen Shot 2020-07-03 at 10 22 02 AM
Screen Shot 2020-07-03 at 10 20 36 AM
</p>

### CODE REVIEW

File: cfgov/unprocessed/apps/mmt-my-money-calendar/js/views/calendar/index/day.js [hunk 1]
```diff
@@ -38,6 +38,7 @@ function Day({ day, dateFormat = 'D' }) {
   if (!eventStore.events.length) return emptyTile(classes);
 
   const balance = eventStore.getBalanceForDate(day);
+  console.log('week ending balance', uiStore.weekEndingBalanceText);
```
Comment (BarakStout) [inline cfgov/unprocessed/apps/mmt-my-money-calendar/js/views/calendar/index/day.js:41]: Wild `console.log`. Please remove.

File: cfgov/unprocessed/apps/mmt-my-money-calendar/js/views/more/index.js [hunk 2]
```diff
@@ -14,47 +14,38 @@ function More() {
   const history = useHistory();
   const [clearDataModalOpen, toggleClearDataModal] = useToggle(false);
 
-  const clearAllData = useCallback(async (evt) => {
-    evt.preventDefault();
-    await eventStore.clearAllData();
-    history.push('/');
-  }, [eventStore, history]);
+  const clearAllData = useCallback(
```
Comment (jbevis) [inline cfgov/unprocessed/apps/mmt-my-money-calendar/js/views/more/index.js:17]: This is knit picky, but I would recommend reverting this back to how it was before.

Breaking a function to a new line for the arguments tends to not fall in line with JS formatting/syntax. It doesn't break anything necessarily but maintains consistency in code style.

Also, I have updated this function for my issue, and this unnecessary change will cause conflicts with that PR.

**Comment by jbevis** [comment]
@MissyM2 I would recommend doing a rebase of ```my-money-calendar``` onto this branch.

Some of your changes are formatting issues, that I'm not sure you intended to make, but could just be that you don't have the most recent changes.

After rebasing locally, push this branch back up and it should take care of those.

File: cfgov/unprocessed/apps/mmt-my-money-calendar/js/views/more/index.js (hunk 2 above)
Comment (MissyM2) [reply cfgov/unprocessed/apps/mmt-my-money-calendar/js/views/more/index.js:17]: @jbevis I am not sure what you mean.  I did not change this.   Could we discuss?

**Comment by jbevis** [comment]
Looks good to me.

Merging.
